python/dgl/graphbolt/data_block.py

```diff
@@ -84,6 +84,7 @@ def to_dgl_graphs(self):
                         edge_type,
                         feature_name,
                     ), feature in edge_feature.items():
+                        edge_type = tuple(edge_type.split(":"))
```

Comment:
use etype_tuple_to_str, added in https://github.com/dmlc/dgl/pull/6230

Reply:
So tuple is used in DGLGraph?

Reply:
yes